airgun/browser.py

```diff
@@ -236,11 +238,30 @@ class AirgunBrowserPlugin(DefaultPlugin):
         }
         '''
 
+    def __init__(self, *args, **kwargs):
+        self._ignore_ensure_page_safe_timeout = False
+        super().__init__(*args, **kwargs)
+
+    @property
+    def ignore_ensure_page_safe_timeout(self):
+        return self._ignore_ensure_page_safe_timeout
+
+    @ignore_ensure_page_safe_timeout.setter
+    def ignore_ensure_page_safe_timeout(self, value):
+        self._ignore_ensure_page_safe_timeout = value
+
     def ensure_page_safe(self, timeout='30s'):
         """Ensures page is fully loaded.
         Default timeout was 10s, this changes it to 30s.
+        If self.ignore_ensure_page_safe_timeout is True, the function doesn't raise an exception
+        and continues as if the page was safe instead. This can be used to bypass
+        some bugs, e.g. https://bugzilla.redhat.com/show_bug.cgi?id=2106022
         """
-        super().ensure_page_safe(timeout)
+        try:
+            super().ensure_page_safe()
```

Comment:
I somehow dropped the timeout parameter from a function all. This will need to be fixed in a separate PR.